ResetPasswordBundle/src/PasswordResetHelper.php

```diff
@@ -150,4 +183,10 @@ private function hasUserHisThrottling(object $user): bool
 
         return false;
     }
+
+    private function getNewExpiresAt(): \DateTimeImmutable
+    {
+        return (new \DateTimeImmutable('now'))
+            ->modify(sprintf('+%d seconds', $this->resetRequestLifetime));
+    }
```

Comment:
When simple logic is isolated into a private method, I think it gets a bit *less* readable. I'd remove this and just do it inline in the original method.

Reply:
done